### ISSUE TITLE
[Misc] Tooltips der Geschwindigkeitstooltips sollten auf Tasten hinweisen

### ISSUE THREAD

**Issue by GWRon** [TVTower/TVTower#756]
Dalli per Discord:
> nur ... als Quick-Tipp könntet ihr die Shortcuts mit einbasteln
> also Geschwindigkeitsbuttons
> suche jedes mal neu xD

Ich:
> du meinst die Tooltips der Buttons sollen auf den Tastenshortcut hinweisen ("Taste 7") ?

Sprich: Die Tooltips der "Interface-Buttons" zur Presetauswahl koennten auf die nutzbaren Tastenkuerzel hinweisen (betrifft auch Escape-Menue und Co).

**Comment by GWRon** [comment]
> und nen Pausebutton 😉
> also wenn schon dennschon 😄
> MadTV hatte sowas auch
> nein, haste auf die Uhrzeit geklickt und hattest pause

Eventuell sollten wir ueber die "Pause" nachdenken...

**Comment by nittka** [comment]
Tooltips für die Geschwindigkeitsbuttons (und Menü und Hilfe) sind keine schlechte Idee. Sinnvoll wäre eigentlich auch, die Kürzel für die Räume irgendwo im Spiel (außer im Handbuch) verfügbar zu haben. Ich dachte da ja mal an den Raumplan (in Klammern hinter dem Namen).

Pause war ja im Forum schonmal ein längeres Thema. Damals war die Aussage: Escape-Menü verhält sich exakt wie Pause.
Ich hatte mal kurz probiert analog Dev-Modus Geschwindigkeit 0 (über Taste 0) auch im produktiven Modus zu nutzen. Das Problem ist, dass man dann trotzdem weiter im Spiel agieren kann (Programmplanung etc,). Das wäre ja nicht Sinn und Zweck der Pausefunktion.

... Hilfe und Menü verhalten sich schon wie Pause. Man könnte also auch einfachd den Tooltip hier auf "Hilfe/Pause" bzw. "Menü/Pause" anpassen.

**Comment by GWRon** [comment]
Ich kannte die Pausefunktion in Mad TV nicht, weiss ne, wie sie dort genau funktionierte.

**Comment by GWRon** [comment]
_Dalli per Discord_:
> Thema Pause:
> Also entweder Taste "P" für Pause alternativ auf die Uhrzeit klicken -> Bild wird abgedunkelt (nicht schwarz) und in der Mitte steht dann Pause (also wie ESC nur ohne Menü)
>
> Oder ESC und P gehen ins Menü, dann schreibt ihr halt Pausenmenü oder sowas drüber. (Taste P für Pause ist halt quasi gängig und logisch) Uhrzeit-Klick könnte dann natürlich gleiche Wirkung haben 😉

**Comment by GWRon** [comment]
Wir koennten also dem Escape-Menue einfach noch einen Titel spendieren:
"Spiel pausiert..."
und vielleicht den Bildschirm etwas abdunkeln (komplett) um die "keine Interaktionsmoeglichkeit"-Sache zu verdeutlichen.

**Comment by nittka** [comment]
Ist so z.T. schon im PR umgesetzt. P als Taste geht nicht, da schon für den Raumplan verwendet.

**Comment by GWRon** [comment]
Ja, mir reicht ja die "Escape-Taste" auch. Ist ja gut zu finden.
Das das Escapemenue auf "pausiert" hinweist, wuerde zumindest nicht schaden.
